### PR TITLE
Create test extractors that check functionality of v2 endpoints

### PR DESCRIPTION
**Pull request by max-zilla**
This is from @ddey2 but was merged into a branch that never got a PR created for it.

### CODE REVIEW

**Comment by tcnichol** [comment]
I can mark this approved, as this would be another issue I checked out before.

**Review by tcnichol — APPROVED** [review]
This one looks good, same as the one from before. Marking approved.

**Comment by ddey2** [comment]
test-file-extractor works.

test-dataset-extractor doesn't work. I can't trigger the extractor on dataset. It gets error on clowder backend.

**Comment by ddey2** [comment]
Screenshot 2023-08-10 at 9 26 02 AM

**Comment by ddey2** [comment]
I have made some fixes on clowder backend to make this working. https://github.com/clowder-framework/clowder2/pull/640

Now, the test-dataset-extractor works. The upload_to_dataset method doesn't work. I get the following error on pyclowder and clowder backend. Still debugging this.
Screenshot 2023-08-15 at 10 11 36 PM
Screenshot 2023-08-15 at 10 11 43 PM

File: sample-extractors/test-dataset-extractor/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pyclowder==2.6.0
```
Comment (longshuicy) [inline sample-extractors/test-dataset-extractor/requirements.txt:1]: The latest pyclowder is 3.0.2?
https://pypi.org/project/pyclowder/

File: sample-extractors/test-file-extractor/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pyclowder==2.6.0
```
Comment (longshuicy) [inline sample-extractors/test-file-extractor/requirements.txt:1]: Same here. Should it be 3.0.2?

**Review by longshuicy — APPROVED** [review]

**Pull request merged**